ENIGMAsystem/SHELL/Makefile

```diff
@@ -114,6 +115,10 @@ else
 	RESOURCEBINARY := $(OBJDIR)/resources.res
 endif
 
+SOURCES += $(addprefix ../shared/,$(SHARED_SOURCES))
```

Comment:
I should have made this say `$(SHARED_SRC_DIR)` instead of `../shared/`. That's what's breaking this makefile; you can change it in the other makefile, too, if you like.